### DISCUSSION TITLE
Feature's version for OSM data

### DISCUSSION THREAD

**Discussion by n-kaytser** [OvertureMaps/data#170] [Discussion]
### Category Feedback

Greetings!
I have a question regarding OSM data.
The problem is that almost all features (except for the base and building themes) are written without a version. Maybe you could tell me why this happens and is it possible to find information about the versions of these features somewhere?
Thank you for your work!
<img width="675" alt="Снимок экрана 2024-06-07 в 15 09 06" src="https://github.com/OvertureMaps/data/assets/78761451/cd854059-bdaf-402e-be08-6b58ce8898d1">
<img width="551" alt="Снимок экрана 2024-06-07 в 15 12 01" src="https://github.com/OvertureMaps/data/assets/78761451/31f523a4-1ac9-48ba-bae6-df749f3a2335">

### Dependency with other categories, if any.

Data Platform

**Comment by jenningsanderson** [comment]
Thanks for the feedback @n-kaytser. 

This is an open issue we're looking to standardize in upcoming releases — we agree, having the OSM version available within the sources is desired.

**Comment by n-kaytser** [comment]
Greetings!
I would like to clarify about the feature versions. Maybe you already know when approximately you plan to add them to the release?
Thank you very much for your answer😊

**Reply by n-kaytser** [reply]
cc @jenningsanderson

**Reply by jenningsanderson** [reply]
Hi @n-kaytser — Yes, this is definitely still on the roadmap of desired improvements, but unfortunately, I don't have an estimate for you.
